openstudiocore/resources/osversion/1_10_4/OpenStudio.idd

```diff
@@ -8276,9 +8276,14 @@ OS:AirLoopHVAC,
        \type object-list
        \required-field
        \object-list ConnectionNames
-  A14; \field Return Air Bypass Flow Temperature Setpoint Schedule Name
+  A14, \field Return Air Bypass Flow Temperature Setpoint Schedule Name
```

Comment:
Hmm, I'm not sure @kbenne is gonna go for that.  I'll get out of the way and let you guys discuss.  @asparke2 should also chime in.